src/Provider/FreeGeoIp/FreeGeoIp.php

```diff
@@ -50,6 +70,10 @@ public function geocodeQuery(GeocodeQuery $query): Collection
             ])]);
         }
 
+        if ($locale = $query->getLocale()) {
```

Comment:
We should not change the providers locale each time we do a geocode lookup. That will make the provider stateful.